ansible/vars-bvt.yaml

```diff
@@ -12,6 +12,6 @@ avocado_conf: "{{avocado_conf_dir}}/avocado.conf"
 test_providers_dir: "{{avocado_data_dir}}/avocado-vt/test-providers.d/downloads"
 
 avocado_repos:
-  - { repo: "https://github.com/autotest/tp-libvirt.git", dest: "{{test_providers_dir}}/io-github-autotest-libvirt" }
-  - { repo: "https://github.com/autotest/tp-qemu.git", dest: "{{test_providers_dir}}/io-github-autotest-qemu" }
-  - { repo: "https://github.com/spiceqa/tp-spice.git", dest: "{{test_providers_dir}}/io-github-spiceqa-spice" }
+  - { repo: "https://{{pipeline_constants.GITHUB_REPOS_BASE_URL}}/autotest/tp-libvirt.git", dest: "{{test_providers_dir}}/io-github-autotest-libvirt" }
```

Comment:
This is a bit confusing. "base url" looks like it would contain the _https://_ part of the URL too. What about `GITHUB_DOMAIN`?